packages/database/schema.prisma

```diff
@@ -577,3 +584,21 @@ model ShortUrl {
   updatedAt DateTime @updatedAt @map(name: "updated_at")
   url       String   @unique
 }
+
+model UserSegment {
```

Comment:
would it be posisble to rename `UserSegment` to `Segment` since we use the `User` term currently for Formbricks users and not `people`